js/code.js

```diff
@@ -184,13 +166,62 @@ function doLogout()
 	window.location.href = "index.html";
 }
 
-function doGoToUpdate()
+function doGoToUpdateContact(string, el)
```

Comment:
This is the new doGoToUpdate(string, el) function it's purpose is to save any needed Contact info to window.SessionStorage to use in the UpdateContact() page.